### PR TITLE
Updated scholar data

### PR DESCRIPTION
**Pull request by chenhajaj**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -17046,10 +17046,10 @@ eva Kühn,TU Wien,http://www.complang.tuwien.ac.at/eva,NOSCHOLARPAGE
...
-Chen Hajaj,Ariel University,https://www.google.com/url?sa=t&source=web&cd=1&cad=rja&uact=8&ved=2ahUKEwjFgdSqhrniAhXN1qQKHUVdDzwQFjAAegQIARAB&url=https%3A%2F%2Fwww.ariel.ac.il%2Fwp%2Fchen-hajaj%2F&usg=AOvVaw01oGV2byT9wxOMYjET6x9O,
-Shani Alkoby,Ariel University,https://www.ariel.ac.il/Projects/trp/GeneralInformation.asp?numRec=687&id_lang=2,
-Erel Segal-Halevi,Ariel University,http://www.ariel.ac.il/Projects/trp/GeneralInformation.asp?numRec=614&id_lang=1,
-Gil Ben Arzi,Ariel University,http://www.ariel.ac.il/Projects/trp/GeneralInformation.asp?numRec=645&id_lang=1,
-Tal Grinshoun,Ariel University,https://www.ariel.ac.il/Projects/trp/GeneralInformation.asp?numRec=426&id_lang=2,
-Lihi Deri,Ariel University,https://www.ariel.ac.il/Projects/trp/GeneralInformation.asp?numRec=489&id_lang=2,
-Inon Zukerman,Ariel University,https://www.ariel.ac.il/Projects/trp/GeneralInformation.asp?numRec=316&id_lang=2,
\ No newline at end of file
+Chen Hajaj,Ariel University,https://www.google.com/url?sa=t&source=web&cd=1&cad=rja&uact=8&ved=2ahUKEwjFgdSqhrniAhXN1qQKHUVdDzwQFjAAegQIARAB&url=https%3A%2F%2Fwww.ariel.ac.il%2Fwp%2Fchen-hajaj%2F&usg=AOvVaw01oGV2byT9wxOMYjET6x9O,Zy2cIskAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:17049]: Please fix this home page: it should not be a google link.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 1 above)
Comment (chenhajaj) [reply csrankings.csv:17049]: My bad, it should be
https://www.ariel.ac.il/wp/chen-hajaj/

Updated the file

**Comment by chenhajaj** [comment]
Can you please direct me on how to resolve this issue?

**Comment by emeryberger** [comment]
It's not clear to me that you meet the inclusion criteria. Do you have a cross-appointment in CS?

**Comment by chenhajaj** [comment]
I have a cross-appointment in the Cyber and Innovation research center, which is under the CS department.
I am also the head of the university's Data Science and AI research center.

**Comment by emeryberger** [comment]
Added manually.

**Pull request closed without merging**